### DISCUSSION TITLE
Deepin23缺少aapt依赖包

### DISCUSSION THREAD

**Discussion by gfdgd-xi** [linuxdeepin/developer-center#4396] [General | 常规讨论]
最新的beta缺失aapt依赖包导致UEngine运行器无法安装

**Comment by BLumia** [comment]
有兴趣参与打包吗？欢迎参与新软件包的维护 :) [参考](https://github.com/linuxdeepin/developer-center/wiki/Become-a-Package-Maintainer-(zh_CN))

**Reply by gfdgd-xi** [reply]
我试试
